assignment.js

```diff
@@ -66,6 +72,18 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+let values = [3, 48, 29, 12, 18]
+let LargestNumber = values[0]
```

Comment:
@justfinish best practice is to use camelCase when naming variables. There are few caveats to this rule but generally, you'll want to default to camelCase when naming your variables.